build_tools/github_actions/runner/README.md

```diff
@@ -121,6 +121,29 @@ release, the version of the program running on the runners must be updated
 otherwise the GitHub control plane will refuse their connection. Testing and
 rolling out these updates involves a few steps.
 
+### MIG Rolling Updates
+
+See https://cloud.google.com/compute/docs/instance-groups/updating-migs for the
+main documentation. There are two modes for a roling MIG update, "proactive" and
```

Comment:
typo (might want get you over the character limit for this line though, if you care)

```suggestion
main documentation. There are two modes for a rolling MIG update, "proactive" and
```

Reply:
:crying_cat_face: autosubmit